jobs/focus_android_events_to_amplitude.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
This shouldn't be merged at any rate until https://github.com/mozilla-services/cloudops-deployment/pull/1799 is deployed which is unlikely to happen today.

Reply:
Ah well, looks like the quarter will have to wait.

@sunahsuh the reason I didn't use the runner is because of the special case of setting the API key. It seems a bit special-casy to add that to the runner.

Reply:
I think if we run into a need like this again, we should consider adding pre-submit hooks to the runner, but this feels like one of those "refactor out if you run into it twice" kinds of things